fastlane/Fastfile

```diff
@@ -425,11 +471,22 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Upload Build to Play Store"
   lane :upload_build_to_play_store do | options |
-    aab_file_path = bundle_file_path(options[:version])
+
+    app = options[:app]
+    package_name = app.downcase == "jetpack" ? JP_PACKAGE_NAME : WP_PACKAGE_NAME
```

Comment:
This is totally ok to merge as is to get this PR land ASAP, but as an improvement suggestion for a future/separate PR: 

A more Ruby-y way of doing this is to use a Hash for those constants. For example, change the constants you added [at the top of the Fastfile here](https://github.com/wordpress-mobile/WordPress-Android/pull/14613/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R71-R80) to something like:

```
PACKAGE_NAME = { wordpress: 'org.wordpress.android', jetpack: 'com.jetpack.android' }.freeze
PREFIX = { wordpress: 'wpandroid', jetpack: 'jpandroid' }.freeze
APP_NAME = { wordpress: 'WordPress', jetpack: 'Jetpack' }.freeze
```
And then replace every place in the code where you use this kind of `downcase == 'jetpack'` test with something like this instead:
```suggestion
    package_name = PACKAGE_NAME[app.downcase.to_sym]
```

Reply:
Opened https://github.com/wordpress-mobile/WordPress-Android/issues/14874 to track this!